holidays/countries/bermuda.py

```
@@ -0,0 +1,141 @@
+#  holidays
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)